### PR TITLE
Enable spark_local in Travis

### PR DESCRIPTION
**Pull request by drostron**

### CODE REVIEW

**Comment by drostron** [comment]
Pending #1922

File: .travis.yml [hunk 1]
```diff
@@ -58,6 +59,7 @@ env:
     - QUASAR_COUCHBASE='"couchbase://0.0.0.0?username=Administrator&password=password"'
     - QUASAR_MARKLOGIC_JSON='"xcc://marklogic:marklogic@localhost:8000/Documents?format=json"'
     - QUASAR_MARKLOGIC_XML='"xcc://marklogic:marklogic@localhost:8000/Documents?format=xml"'
+    - QUASAR_SPARK_LOCAL='local[*]|/'
```
Comment (rabbitonweb) [inline .travis.yml]: I get now, LGMT

**Comment by rabbitonweb** [comment]
Part of https://github.com/quasar-analytics/quasar/issues/1666

**Comment by rabbitonweb** [comment]
Spark Connector `spark-local` is running on Travis! This covers majority of possible errors for all sparkcore dependent connectors

**Pull request merged**